# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../musl/pkg/gdb/gdb-76-pid_t.patch
# Copyright (C) 2014 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

This is a patch from sabotage-linux
https://github.com/sabotage-linux/sabotage/blob/master/KEEP/gdb-76-pid_t.patch
--- gdb-7.6.2.org/gdb/common/linux-ptrace.h
+++ gdb-7.6.2/gdb/common/linux-ptrace.h
@@ -67,6 +67,7 @@
 #define __WALL          0x40000000 /* Wait for any child.  */
 #endif
 
+#include <unistd.h> /* for pid_t */
 extern void linux_ptrace_attach_warnings (pid_t pid, struct buffer *buffer);
 extern void linux_ptrace_init_warnings (void);
 
